.atomist/apache2.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
If you want to keep these and make it extensible, then I'd suggest creating a subdirectory where the license files and header snippets map to the parameter value.